### ISSUE TITLE
Error : Multi class Multi output is not supported

### ISSUE THREAD

**Issue by cassinthangam4996** [huggingface/setfit#347]
Hi, 

I am training a multi-label classification dataset and I am getting an error in the evaluate() step. I had previously trained the same dataset a few months before and never faced any issues. 

![image](https://user-images.githubusercontent.com/22761661/231277456-bc34b7db-24c9-42ef-8643-9902c16f89aa.png)

**Comment by tomaarsen** [comment]
The final error comes from sklearn, here to be exact:
https://github.com/scikit-learn/scikit-learn/blob/c8fb5611823a878cf96236064b0f08dc4f5c5211/sklearn/metrics/tests/test_classification.py#L540

I haven't seen this myself before. Could you maybe provide a more complete example with toy data that also experiences this issue?

- Tom Aarsen

**Comment by tomaarsen** [comment]
Alternatively, you can specify your own `compute_metrics` function, and directly try to use `accuracy_score` from sklearn, or some other metric that you want.
